js/exmo.js

```diff
@@ -1758,22 +1758,22 @@ module.exports = class exmo extends Exchange {
         // fixed funding fees only (for now)
         if (!this.fees['transaction']['percentage']) {
             const key = (type === 'withdrawal') ? 'withdraw' : 'deposit';
-            let feeCost = this.safeNumber (transaction, 'commission');
+            let feeCost = this.safeString (transaction, 'commission');
             if (feeCost === undefined) {
-                feeCost = this.safeNumber (this.options['transactionFees'][key], code);
```

Comment:
`fetchDeposits` works, however `fetchTransactions` throws an error when there is no transaction fee for this code.

This error is unrelated to this change, and the change you propose here I think is correct. I'll open a different PR to fix it.

Reply:
thanks

Reply:
@pcriadoperez Could you actually separate stex and exmo into separate PRs?